src/Components/CardList.tsx

```diff
@@ -80,14 +90,88 @@ export const CardList = ({
     if (item?.format && formats.includes(item?.format)) return true
   }
 
+  function genderFilter(
+    item: ServicePreview,
+    genderInput: Array<string> | undefined
```

Comment:
```ts
genderInput?: Array<string>
```

would be a more Typescripty way of writing this, but it's not wrong.

Reply:
This is true 👍